### PR TITLE
Melodic devel staging

### PR DESCRIPTION
**Pull request by fmauch**
I believe that melodic-devel-staging is elaborate enough to be merged into melodic-devel.

### CODE REVIEW

File: melodic.ignored [hunk 1]
```diff
@@ -0,0 +1 @@
+ur_kinematics
```
Comment (gavanderhoorn) [inline melodic.ignored]: Wouldn't this have to be added to the release repository?

Comment (fmauch) [reply melodic.ignored]: I was going after https://answers.ros.org/question/66545/to-ignore-some-pkgs-upon-release-by-bloom/?answer=66548#post-id-66548 but you might be right. I've never done that.

**Review by RobertWilbrandt — APPROVED** [review]
With #619 merged all breaking changes should now be done and there is no more reason to delay a release.

**Comment by fmauch** [comment]
Not deleting the source branch, since there are many references on that.

**Pull request merged**